LASlib/src/LASlibConfig.cmake

```diff
@@ -0,0 +1,8 @@
+include("${CMAKE_CURRENT_LIST_DIR}/LASlibTargets.cmake")
+add_library(LAStools::LASlib INTERFACE IMPORTED)
+target_link_libraries(LAStools::LASlib INTERFACE LASlib)
```

Comment:
I would strongly recommend to turn it around: Export with namespace, create legacy name as alias.

Reason: The namespaced name shall be used when passing on transitive usage requirements, to enable the CMake configuration diagnostics which are the motivation of this PR.

Reply:
ok